source/Examples/SmartCache/README.md

```diff
@@ -75,7 +75,8 @@ To build and run the necessary components for this example
 
 ## Executing
 
-The following commands will start the `Mlos.Server.Agent` and cause it to start the `SmartCache` component microbenchmark:
+The following commands will start the `Mlos.Server.Agent` and cause it to start the `SmartCache` component microbenchmark. We're also passing
+an `optimizer-uri`, which requires us to start an optimizer microservice (``start_optimizer_microservice launch``).
```

Comment:
Either all of that, or we just drop the`--optimizer-uri` line from the example command below, and add a comment to refer people to the SmartCacheCPP notebook for a demonstration of using the example with an optimizer.

Reply:
I'm slightly worried about the duplication but I'm not sure which place is the better place for it.
I hope to not address the PATH issue, it won't come up when using conda and we should use conda everywhere. We don't use it inside the container right now but it would probably simplify things.

For now I think linking to the python setup might be easiest, so people will use conda and won't run into that issue (or any of the compilation issues).

Reply:
I forgot this is inside the docker container, so the pip install should be fine.

Reply:
Most of the explanation text for this one has already been put in the notebook, right?

This was initially created as a starting point for that anyways.  Why don't we just change this be a mostly empty file with a link to that notebook instead, and call it done.  That way we still have a landing page link to something with some more details in the github view and we aren't duplicating descriptions.

Sound good?

Reply:
There were some annoyances with getting the links right (the rendered notebook doesn't exist on github pages yet).
Will take that up in a future PR just to move us along for now.

Reply:
https://github.com/microsoft/MLOS/pull/123